util/find-doc-nits

```diff
@@ -601,6 +601,13 @@ sub wording {
     }
     err($id, "found 'epoch' should use 'Epoch'")
         if $contents =~ /\bepoch\b/;
+    if ( $id =~ m@man1/@ ) {
+        err($id, "found 'tool' in NAME, should use 'command'")
+            if $contents =~ /=head1 NAME.*tool.*=head1 SYNOPSIS/s;
```

Comment:
Might I suggest this regexp?  `/=head1 NAME.*\btool\b.*=head1 SYNOPSIS/`

`\b` matches the boundary between word and non-word

Reply:
Worried someone might put, "stool" in the name? :)  Sure.